arm/Microsoft.Compute/galleries/.parameters/images.parameters.json

```
@@ -3,7 +3,7 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "name": {
-            "value": "sxxazsigweux002"
+            "value": "sxxazsigweuimg002"
```

```suggestion
            "value": "sxxazsigweuimages001"
```

or rename the parameter file to img.parameters.json